Syllabi/misc.md

```diff
@@ -0,0 +1,56 @@
+# Miscellaneous thoughts on assessment
+
+## What should we teach?
+
+* We've mostly spoken about software tools (i.e., Git, GitHub, Docker,
+  Python, R, Make, unit tests, etc.) for reproducibility.  Yet the mechanical
+  reproduction of results is not an end, but a means to provide a secure
+  foundation for reliable research.  To what extent
+  should we be teaching/discussing other aspects of computational
+  reproducibility (e.g., statistical issues) and tool-agnostic principles
+  that can be introduced in many different contexts.
```

Comment:
Teaching some philosophy of science might increase students' 'buy-in', convincing them that reproducibility is a thing to pursue. 

Relatedly: there might be an argument for teaching "data science for future presidents" (in analogy to the UCB "physics for future presidents"), in which the tools are not taught, but the concepts, and their importance, are.